src/cpp/benders/benders_core/BendersBase.cpp

```diff
@@ -679,6 +659,65 @@ void BendersBase::compute_cut_aggregate(const SubProblemDataMap& subproblem_data
     _master->add_cut(s, _data.x_cut, rhs);
 }
 
+
+void BendersBase::build_all_aggregated_cuts(const std::vector<std::vector<std::pair<std::string,int>>>& subproblem_names, const std::vector<SubProblemDataMap>& gathered_subproblem_map)
+{   
+    std::vector<int> subproblem_ids_per_cut ; 
+    for (const auto& subproblem_names_in_cut : subproblem_names) 
+    {
+        Point s; 
+        double rhs(0) ; 
```

Comment:
nitpick: Using curly braces avoid confusion with function definition
```suggestion
        double rhs{0} ; 
```